bluesky/run_engine.py

```diff
@@ -2707,6 +2722,21 @@ def _default_md_validator(md):
             "BAD: sample=[1, 2] ")
 
 
+def _default_md_normalizer(md):
```

Comment:
The validator has this handled, so I think we the default normalizer can just be a no-op, i.e. `return md`. The copying should happen first, agreeing with @dmgav's comment above.

Reply:
My idea was that copying should not be delegated to whoever implements the normalizer.

Reply:
Yeah, we are agreeing.

I meant that the sample validation below here is already handled by the default validator and need not be copied into the normalizer also. Sorry, I wrote this in an unclear way.